### ISSUE TITLE
Make 'custom build profiles' testable and add tests for each

### ISSUE THREAD

**Issue by rhusar** [clusterbench/clusterbench#268]
Since we are using maven to build custom configurations of clusterbench to be used in different tests, these are very easily broken with the default build.

Perhaps we need to redesign these profiles a bit, e.g. to add an execution + suffix (?) and add tests https://github.com/clusterbench/clusterbench#custom-build-profiles for each.

**Comment by traitsisgiorgos** [comment]
is the issue still open?

**Comment by rhusar** [comment]
Yes, but https://github.com/clusterbench/clusterbench/issues/354 needs to be done first, so that in a single run all the variants are built first.
